samples/spring-customer-registry-views-quickstart/src/main/java/customer/api/Address.java

```diff
@@ -0,0 +1,18 @@
+package customer.api;
+/*
+ * Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Address {
+  public String street;
+  public String city;
+
+  @JsonCreator
+  public Address(@JsonProperty("street") String street, @JsonProperty("city") String city) {
```

Comment:
I think this Java compiler flag avoids having to repeat the property names: https://github.com/akka/akka-platform-guide/blob/main/docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-java/pom.xml#L175

Reply:
Possible, I even think that `@JsonProperty` should not be needed at all. If I'm not mistaking there was something related with Spring itself. I think I wrote it in a commnet somewhere. I will check it.